android-kit-base/src/test/kotlin/com/mparticle/kits/KitConfigurationTest.kt

```diff
@@ -571,7 +585,7 @@ class KitConfigurationTest {
         consentForwardingRule.put("v", ruleArray)
         val rule1 = JSONObject()
         ruleArray.put(rule1)
-        rule1.put("h", KitUtils.hashForFiltering("1" + "foo purpose"))
+        rule1.put("h", HashingUtility.hashGDPRContentPurposeKey("foo purpose"))
```

Comment:
For example here the `"1" +` was removed...

In that case, why did the previous error not cause a test failure? Are we not actually testing that data? If not, why include it in the test?